src/components/map/Map.tsx

```diff
@@ -331,18 +331,16 @@ export const Map = ({
                 />
               )}
 
-              {calloutTextEnabled && (
+              {calloutTextEnabled && (!!title || !!serviceName) && (
```

Comment:
as the service name is rendered before the title, the check should be the same for better reaability and context